core/src/main/java/org/frankframework/soap/SoapWrapper.java

```diff
@@ -168,6 +167,10 @@ private String extractMessageWithTransformers(TransformerPool transformerS11, Tr
 		String extractedMessage;
 		if (soapVersion == SoapVersion.SOAP12) {
 			extractedMessage = transformerS12.transform(message.asSource());
+			// If session had the wrong SOAP version stored (e.g. multiple SoapWrappers), try SOAP 1.1 too.
```

Comment:
This works for now but for future I think we should find a better solution.
Can you please add a TODO comment for that in the code, at least in master?
(we don't need such TODO in the bugfix branches).